03-rnaseq/pathway-analysis_rnaseq_03_gsva.Rmd

```diff
@@ -0,0 +1,796 @@
+---
+title: "Gene set variation analysis - RNA-seq"
+author: "CCDL for ALSF"
+date: "November 2020"
+output:  
+  html_notebook:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+**IDENTICAL TO MICROARRAY GSVA** (except microarray -> rna-seq)
+
+# Purpose of this analysis
```

Comment:
I might not touch this until we see what happens for #371

Reply:
That seems reasonable.